object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,10 @@
+/* exported toObject */
+function toObject(keyValuePair) {
+  let object = {};
```

Comment:
looks like you're not using this either, the way this function is written you may as well just declare `const object` on line 9, or just return the object literal you created